Netezos/Forging/Local/LocalForge.Forgers.cs

```diff
@@ -0,0 +1,211 @@
+﻿using Netezos.Encoding;
```

Comment:
I just moved this to a new partial file since I wanted to put `unforging` operations in a counterpart file.